api/cpp/include/slint.h

```diff
@@ -1318,6 +1318,16 @@ inline void update_all_translations()
 }
 #endif
 
+/// Select the current translation language when using bundled translations.
+/// This function only has effect if the slint code was compiled with bundled translations.
```

Comment:
```suggestion
/// This function requires that the application's `.slint` file was compiled with bundled translations.
```

(to disambiguate slint the library vs. the application's slint file)